Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -735,6 +798,8 @@ - (void) testReplicatorAndListenerOnSameDatabase {
     repl2 = nil;
     Assert([db2 close: &err], @"Failed to close db2 %@", err);
     db2 = nil;
+    
+    [self stopListen];
```

Comment:
teardown calls same?
```suggestion
```

Reply:
Yes, just in case a test forgets to stop listen.